cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/operationrunner/InMemorySourceControlOperationRunner.java

```diff
@@ -228,7 +228,10 @@ public RepositoryAppsResponse list(NamespaceId namespace, RepositoryConfig repoC
       for (File configFile : configFiles) {
         String applicationName = FileUtils.getNameWithoutExtension(configFile.getName());
         try {
-          String fileHash = repositoryManager.getFileHash(basePath.relativize(configFile.toPath()), currentCommit);
+          String fileHash = repositoryManager.getFileHash(
```

Comment:
This looks a bit odd that we're calling `repositoryManager. getFileRelativePath` inside `repositoryManager.getFileHash`, could we consolidate them if possible?

Reply:
`repositoryManager.getFileHash` takes relative path to git repo which is correct IMHO. I have separated `repositoryManager.getFileRelativePath` out as the path should be reused in logs anyway